adopted-ember-addons/ember-tracked-local-storage
# [#16417] Remove unnecessary format functionality

Closes https://github.com/Leadfeeder/issue-tracker/issues/16417

It's not necessary to define a format for the value, as we can just JSON stringify / parse the value and it will work properly.

This PR removes the normalize from string util, and changes the logic so the tracked local storage class will stringify the value being set in browser local storage, then parse it when retrieving the value.

(It also adds the `keyNameProperty` feature which we are currently using in the monorepo package)

### How to test

You can fetch and run the PR locally, then grabbing the `tracked-local-storage` service from ember inspector, and try manually setting / getting some local storage props of different types, and make sure everything is working properly.
